Automattic/jetpack
# VideoPress: add updateVideoPrivacy() action

This PR adds a sync action to update the video settings.


#### Changes proposed in this Pull Request:

* VideoPress: add updateVideoPrivacy() action


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test with the dev console of your browser
* Go to a single video page, for instance `http://localhost/wp-admin/admin.php?page=jetpack-videopress#/video/<video-id>/edit`
* open the dev console in the client
* get the video data
```
wp.data.select( 'videopress/media' ).getVideo ( <video-id> ).privacySetting
```

* Confirm you  get the `0`, `1` or `2` number
* Change the privacy to another number: `0`, `1`, or `2`

```
wp.data.dispatch( 'videopress/media' ).updateVideoPrivacy ( 941, <new-privacy-value> )
```

* Confirm the client performs an async request to update the privacy

<img width="843" alt="Screen Shot 2022-09-23 at 12 07 13" src="https://user-images.githubusercontent.com/77539/191992766-dc43eb99-f35e-4b9b-9269-e34e8af0e675.png">

* Confirm the state is updated

<img width="517" alt="Screen Shot 2022-09-23 at 12 08 52" src="https://user-images.githubusercontent.com/77539/191993068-2f0c2fad-6c95-4ef2-a894-e388ef56684e.png">

* Confirm the data has been updated on the server-side. Hard refresh, hit the endpoint, etc.
